tests/cases/whitespace/method-call

```diff
@@ -3,25 +3,36 @@ Method calls must not have unnecessary whitespace
 ---CONTENTS---
 <?php
 
-$foo -> bar ( $arg1 );
+$foo -> bar ( $arg1 ) -> baz ( );
 
 $foo
 -> bar ($arg1
 ,$arg2)
 -> baz ($arg1
 ,$arg2);
 
+$fooBar()
+    ->baz()
+    ->qux()
+;
+
 ---FIXED---
 <?php
 
-$foo->bar($arg1);
+$foo->bar($arg1)->baz();
+
+$foo
+    ->bar(
+        $arg1,
+        $arg2
+    )
+    ->baz(
+        $arg1,
+        $arg2
+    );
```

Comment:
Unsure about whether the semi-colon should go on the next line. Produces nicer diffs, but looks ugly.

Reply:
Ah so this is enforced, even if you manually put it on the next line.

Reply:
I thought semicolon-on-next-line was a Symfony standard by now

Reply:
I'd favor it not because of the diffs but because keeps lines independent (for fluent interfaces, which is the use case for chained calls): 
- add a new call
- remove a call
- change the order of calls
- swap two calls
become operations that only involve full lines